frontend/stores/list-watch.js

```diff
@@ -0,0 +1,41 @@
+import { observable, action } from 'mobx'
+import { createContext } from 'react'
+import axios from 'axios'
+
+class ListWatchStore {
+  @observable watches = []
+
+  @action
+  async fetchWatches(userId) {
+    try {
+      const response = await axios.get('/api/watch-manager/users/' + userId)
+      if (response.status === 200) {
+        this.watches = response.data
+      }
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  getWatchById(_id) {
```

Comment:
```suggestion
  get(id) {
```
Thường em sẽ không cần nhiều truy vấn phức tạp ở frontend đâu nên cũng ko cần phải để tên method dài thế.